MapboxDirections/MBVoiceInstruction.swift

```diff
@@ -3,30 +3,43 @@ import Foundation
 @objc(MBVoiceInstruction)
 public class VoiceInstruction: NSObject, NSSecureCoding {
     
-    public let distanceAlongGeometry: CLLocationDirection
+    /**
+     The distance in meters along the current `RouteStep` to give the voice announcement.
+     */
+    public let distanceAlongStep: CLLocationDistance
 
-    public let textAnnouncement: String
     
-    public let SSMLAnnouncement: String
+    /**
+     A plain text representation of the voice instruction to give to the user.
+     */
+    public let text: String
+    
+    
+    /**
+     The voice instruction formatted with [SSML](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/speech-synthesis-markup-language-ssml-reference) (Speech Synthesis Markup Language).
+     
+     Note, when speaking an instruction with this property, the speed synthesis engine must support SSML.
```

Comment:
This library should be synthesizer-agnostic, so let’s link to [Wikipedia](https://en.wikipedia.org/wiki/Speech_Synthesis_Markup_Language) instead of the Alexa documentation.

> A formatted representation of the speech-optimized instruction.
> 
> This representation is appropriate for speech synthesizers that support the [Speech Synthesis Markup Language](https://en.wikipedia.org/wiki/Speech_Synthesis_Markup_Language) (SSML), such as [Amazon Polly](https://aws.amazon.com/polly/). Numbers and names are marked up to ensure correct pronunciation. For speech synthesizers that lack SSML support, use the `text` property instead.